prow/cluster/components/30-crier_deployment.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       containers:
         - name: crier
           image: gcr.io/k8s-prow/crier:v20200319-1aea24112
+          env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: "/etc/credentials/service-account.json"
```

Comment:
This file is also passed as an argument, is this needed?

Reply:
Yes, pubsub reporter doesn’t use gcs-credentials-file arg.